### PR TITLE
chore(docs): add aria-current to main side-drawer

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

**Review by yinonov — CHANGES_REQUESTED** [review]
by replicating the same code to both layout pages in a script we ignore a principal of DRY.
perhaps we can create a new js dedicated file for this purpose. I expect it to scale as we might even need to consider some links are disclosed by default and will need to be undisclosed on load

**Comment by rinaok** [comment]
> 

@yinonov 

I added a scrollTop for the sideDrawer, now it saves the last scrollTop in localStorage and know where to scroll.

We don't have disclosures yet so I think that it's fine for now, and when the day will come we can add some more functionalities.

**Comment by yinonov** [comment]
> > 
> 
> 
> 
> @yinonov 
> 
> 
> 
> I've added a scrollTop for the sideDrawer, now it saves the last scrollTop in localStorage and know where to scroll.
> 
> 
> 
> We don't have disclosures yet so I think that it's fine for now, and when the day will come we can add some more functionalities.

Come to think of it, this is a common use case for SPA. Could there be a library servicing this specifically? Maybe we can unload this to a 3rd party

File: apps/docs/assets/scripts/scroll-handler.js [hunk 1]
```diff
@@ -8,6 +8,20 @@ const updateHeaderElevationShadow = (isShadowed) => {
...
+}
+
+const setScrollFromLocalStorage = (target) => {
+	// set sideDrawer's scroll from localStorage
+	let scrollTop = localStorage.getItem("scroll");
+	if (scrollTop != null) {
+		setTimeout(() => {
+			target.scrollTop = scrollTop;
+		}, 1);
+	}
```
Comment (yinonov) [inline apps/docs/assets/scripts/scroll-handler.js:24]: what for?

File: apps/docs/_includes/navigation.njk [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {% block content %}
```
Comment (yinonov) [inline apps/docs/_includes/navigation.njk:1]: does links scroll into view by default with the aria-current? 🤔

Comment (rinaok) [reply apps/docs/_includes/navigation.njk:1]: @yinonov no

File: apps/docs/assets/scripts/scroll-handler.js (hunk 1 above)
Comment (rinaok) [reply apps/docs/assets/scripts/scroll-handler.js:24]: it saves the scroll, I need to change it to scrollIntoView

File: apps/docs/pages/designs.njk [hunk 3]
```diff
@@ -13,4 +13,10 @@ permalink: "designs/{{ design.title | slug }}/"
 {% endfor %}
 
 
-{% renderFile design.markdown %}
\ No newline at end of file
+{% renderFile design.markdown %}
+
+<script>
```
Comment (yinonov) [inline apps/docs/pages/designs.njk:18]: repeating the same code is probably anti-pattern.
you can, alternatively, observe [mutations to the DOM](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver)

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/pages/designs.njk (hunk 3 above)
Comment (yinonov) [reply apps/docs/pages/designs.njk:18]: maybe a less "expensive" implementation would be to simply run query selector whenDefined (`vwc-nav-item`)

Comment (rinaok) [reply apps/docs/pages/designs.njk]: @yinonov added a navigation handler

**Review by rachelbt — APPROVED** [review]
LGTM
Love the fact that we have indication to which component are we seeing in the docs 🥳

**Pull request merged**